content.test.ts

```diff
@@ -87,9 +87,22 @@ describe.each(glob.sync('content/**/*.md'))('%s', (file) => {
     Tags?: string[];
   };
 
-  const { attributes }: FrontMatterResult<FrontMatterAttributes> = frontmatter(
-    fs.readFileSync(file, 'utf8')
-  );
+  const fileContent = fs.readFileSync(file, 'utf8');
+
+  if (!fileContent.startsWith('---')) {
+    throw new Error(
+      `error: ${file} does not contain properly formatted frontmatter. Check for leading invisible characters.`
```

Comment:
Test does work nicely for leading invisible character 👍 
Looks like it also fails if the file lacks any frontmatter at all though...would it be worth adding something like "If you did include the frontmatter at the top, check for leading invisible..." to potentially avoid confusing a new contrib who just didnt include any frontmatter?

Reply:
I believe frontmatter is required in all files but I can make a more helpful message for those who forgot it

Reply:
sorry for the delay, actually I think reordering the two tests I added works just fine here as `frontmatter.test` passes even with a leading byte order mark